### PR TITLE
Move f/g call stats to evaluate()

### PR DESCRIPTION
**Pull request by vladimir-ch**
Review and merge #35 first.

### CODE REVIEW

**Comment by btracey** [comment]
I'll review this when #35 gets merged. The diff is too big right now.

**Comment by vladimir-ch** [comment]
Merged and updated, PTAL @btracey .

File: local.go [hunk 1]
```diff
@@ -328,24 +334,7 @@ func evaluate(funcs functions, funcInfo *FunctionInfo, evalType EvaluationType,
 }
 
 // update updates the stats given the new evaluation
-func update(location *Location, optLoc *Location, stats *Stats, funcInfo *FunctionInfo, evalType EvaluationType, iterType IterationType, startTime time.Time) {
```
Comment (btracey) [inline local.go]: Is there a utility in still having update now that it's trivial? Seems like it may go into evaluate.

Comment (btracey) [inline local.go]: Oh, sorry, the codereview tool cut off some of the function. Makes more sense now.

**Comment by btracey** [comment]
LGTM.

**Pull request merged**